onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -481,6 +492,11 @@ def _where_input_wrangler(
         matcher=lambda sample: isinstance(sample.kwargs.get("padding"), str),
         reason="String padding is not accepted by aten::conv2d",
     ),
+    skip(
```

Comment:
I am trying to follow why this skip is needed. Could you elaborate?

Reply:
in op.SoftmaxCrossEntropyLoss(), we only accept rarget as INT64 type. but the torch's test case can have FLOAT type which means something else.

Reply:
That's curious. What does it mean? Do we need to create an overload to implement the FLOAT case?

Reply:
from the document: 

**target ([Tensor](https://pytorch.org/docs/master/tensors.html#torch.Tensor)) – Ground truth class indices or class probabilities;**

Reply:
I see! I guess we can worry about this later